plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/DisabledGlueColumnStatisticsProvider.java

```diff
@@ -57,9 +57,9 @@ public void updateTableColumnStatistics(Table table, Map<String, HiveColumnStati
     }
 
     @Override
-    public void updatePartitionStatistics(Partition partition, Map<String, HiveColumnStatistics> columnStatistics)
+    public void updatePartitionStatistics(Set<StatisticsUpdate> statisticsUpdates)
     {
-        if (!columnStatistics.isEmpty()) {
+        if (statisticsUpdates.stream().anyMatch(update -> !update.getColumnStatistics().isEmpty())) {
```

Comment:
super duper nit: Converting to `noneMatch` would allow using static reference to the method and prevent the double-negation type thing here.